NuRadioMC/EvtGen/generator.py

```diff
@@ -1026,6 +1026,8 @@ def get_inverse_spectrum(N, gamma):
 
     print("number of fiducial showers", len(data_sets_fiducial['flavors']))
 
+    data_sets_fiducial["event_ids"] = np.arange(len(data_sets_fiducial["event_group_ids"])) + start_event_id
```

Comment:
was this a bug before, or why did you need to add this line?

Reply:
We treat the event_group_ids as what was the previous event_ids data set, since they are the same thing. event_ids represents a new dataset giving unique ids to each interaction vertex. It does not matter what number as long as it is unique. That's why I only assign event_ids to the fiducial events that are going to be simulated afterwards.